doc/options.md

```diff
@@ -122,6 +121,71 @@ can first generate a `.xml` file as described below.
 To inspect the `.xml` file generated by Lambdapi, one may use the following
 dummy command: `--termination "cat > output.trs; echo MAYBE"`.
 
+##### Printing decision trees
```

Comment:
I propose to put all this in a separate file, and add a link on line 22.

Reply:
OK. We should probably change our documentation support to have something more structured than a bunch of markdown files anyway